packages/terra-theme-provider/src/ThemeProvider.jsx

```diff
@@ -1,8 +1,5 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import className from 'classnames';
-
-const cx = className;
 
 const ThemeProviderThemes = {
```

Comment:
While we're in MVB mode, do we want to kill these theme constants? Are we using these anywhere?

Reply:
After a quick search in our internal github, I found that there a couple of places where the "**Consumer** theme constant" is being used : https://github.cerner.com/search?q=ThemeProvider.Opts.Themes&type=Code.
As for the "**Mock** Theme constant", that looks like it is only being used in our test pages.

Reply:
I'd like to remove these constants, they aren't up to date.